GiniVision/Classes/Networking/DocumentServiceProtocol.swift

```diff
@@ -58,8 +58,8 @@ extension DocumentServiceProtocol {
                                                                                cancellationToken: cancellationToken)
             }).continueWith(block: { task in
                 if let createdDocument = task.result as? GINIDocument {
-                    print("📄 Created document with id:", createdDocument.documentId ?? "",
-                          "for vision document:", document.id)
+                    Logger.debug(message: "Created document with id: \(createdDocument.documentId ?? "") " +
+                        "for vision document \(document.id)", event: .custom(emoji:"📄"))
```

Comment:
I think your Event enum breaks a little bit here. Instead of just putting the emoji itself, you need to have a whole statement - `.custom(emoji:"📄")`. It's kind of verbose. Maybe you can find a way to shorten it and have only the character as parameter?